### PR TITLE
Update DEVELOPMENT.md

### LINKED ISSUE #363: Appsody.dev Development guide should include how to test links during development
The Development guide needs improving as it doesn't mention that you need to use `gatsby build && gatsby serve` to test links during docs development.

### PR DESCRIPTION
**Pull request by helenmasters**
Added how to test links within your documentation.

## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Informs users that they need to use `gatsby build && gatsby serve` if they need to test links within their documentation.

## Related Issues
Fixes #363

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ gatsby clean
 
 For more information on adding Markdown pages with Gatsby, see https://www.gatsbyjs.org/docs/adding-markdown-pages/
 
+To test links within your documentation, follow the instructions for [Testing the website ready for release](https://github.com/appsody/website/blob/master/DEVELOPMENT.md#testing-the-website-ready-for-release).
```
Comment (SueChaplain) [inline DEVELOPMENT.md]: Presumably `gatsby build` reports link errors?

Comment (helenmasters) [reply DEVELOPMENT.md]: I've just put a broken link into one of the docs pages, saved it and run `gatsby build`. No errors are reported on that page, however, when I do `gatsby serve` and click on it the link is indeed broken.

Comment (SueChaplain) [reply DEVELOPMENT.md]: Maybe just say "to manually test links within your documentation....."

Comment (helenmasters) [reply DEVELOPMENT.md]: I've pushed the requested changes, please can you rereview @SueChaplain?

**Pull request merged**